Problem: am rpm SS directory typo in Makefile

The `rpm/Makefile` is looking for the `storage-service` directory but it doesn't exist.

For example, running `make all` from the `rpm` directory:

<pre>
make -C ./storage-service
make[1]: *** storage-service: No such file or directory.  Stop.
Makefile:6: recipe for target 'build' failed
make: *** [build] Error 2
</pre>

--- a/rpm/Makefile
+++ b/rpm/Makefile
@@ -4,7 +4,7 @@ all: build createrepo
 
 build:
 	make -C ./siegfried
-	make -C ./storage-service
+	make -C ./archivematica-storage-service
 	make -C ./archivematica
 	make -C ./nailgun
 	make -C ./fits

